sdcm/mgmt/common.py

```diff
@@ -10,6 +10,12 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def get_persistent_snapshots():  # Snapshot sizes (dict keys) are in GB
```

Comment:
shouldn't you be filtering only for the relevant backend the test is running?

Reply:
Again documenting this function would be nice